frontend/src/app/pages/exercises/exercise/shared/core/drag-element.service.ts

```diff
@@ -251,6 +257,12 @@ type TransferTemplate =
     | {
           type: 'vehicle';
           template: VehicleTemplate;
+          personnelTemplates: {
+              [key in PersonnelType]: PersonnelTemplate;
+          };
+          materialTemplates: {
+              [key in MaterialType]: MaterialTemplate;
+          };
```

Comment:
Aren't these the same for all vehicleTemplates? If so, you could just leave this like it was before and get these templates in `createVehicleParameters()`

Reply:
Yes they are always the same. I just interpreted the Phrase the Function gets these things as they should be given to it.

Reply:
But I dont really know how to access the state from that function since it is not part of the frontend. Should I just use the default Values or how do I access the state from there?

Reply:
> But I dont really know how to access the state from that function since it is not part of the frontend. Should I just use the default Values or how do I access the state from there?

I think my previous comment was a bit misleading.

* The `createVehicleParameters` function should stay as it currently is.
* The call of `createVehicleParameters()` in this service should not take the material and personnelTemplates from the dragEvent, but just use the ones from the store. The code for this should be in my comment below.
* You can remove the materialTemplates and personnelTemplates forms that are marked in this comment.